python/lsst/ts/observatory/control/base_camera.py

```diff
@@ -965,7 +965,10 @@ async def take_imgtype(
         if imgtype not in ["BIAS", "DARK"]:
             await self.setup_instrument(**kwargs)
 
-        if imgtype == "OBJECT" and self.ready_to_take_data is not None:
+        if (
+            imgtype in ["OBJECT", "ENGTEST", "ACQ", "CWFS"]
+            and self.ready_to_take_data is not None
+        ):
```

Comment:
A couple line below (line 983) there is another `elif` that needs updating. It warns users that the TCS synchronization feature is not configured. You might want to extract the list into a variable to use in both places, e.g.:

```
sync_imgtypes = ["OBJECT", "ENGTEST", "ACQ", "CWFS"]

if imgtype in sync_imgtypes:
    ...
```